migrate/20251023_vm_init_script.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:vm_init_script) do
+      foreign_key :id, :vm, type: :uuid, primary_key: true
+      column :script, String, null: false
```

Comment:
```suggestion
      column :script, String, size: 2000, null: false
```

pushing the envelope on smallness...and toastyness.